File: test/aidbox_sdk/snapshots/dotnethl7-fhir-r4-core/PaymentNotice.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class PaymentNotice : DomainResource, IResource
{
    public Base.ResourceReference? Response { get; set; }
    public required Base.Money Amount { get; set; }
    public Base.ResourceReference? Request { get; set; }
    public required Base.ResourceReference Payment { get; set; }
    public required Base.ResourceReference Recipient { get; set; }
    public required string Created { get; set; }
    public Base.CodeableConcept? PaymentStatus { get; set; }
    public required string Status { get; set; }
    public Base.ResourceReference? Payee { get; set; }
    public string? PaymentDate { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.ResourceReference? Provider { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/PaymentNotice.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import DomainResource
from base import Money
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class PaymentNotice(DomainResource):
    response: Optional[Reference] = None
    amount: Money
    request: Optional[Reference] = None
    payment: Reference
    recipient: Reference
    created: str
    payment_status: Optional[CodeableConcept] = None
    status: str
    payee: Optional[Reference] = None
    payment_date: Optional[str] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    provider: Optional[Reference] = None